addon/plugins/image/plugins/checkPasteSize.ts

```diff
@@ -27,7 +28,13 @@ export function checkPasteSize({
           if (onLimitReached) {
             onLimitReached();
           } else {
-            console.error('Paste size is bigger than expected');
+            // Show a notification via the notification plugin
+            notificationPluginKey.getState(view.state).notificationCallback({
+              title: 'Paste size limit reached',
```

Comment:
This hardcoded message isn't translated, and it also doesn't explain to the user what they can do about it